repository/repository/controllers.py

```diff
@@ -48,4 +48,28 @@ def get_eprint_events(identifier: str, version: int) -> Response:
     except DoesNotExist as e:
         raise NotFound(f'No such e-print: {identifier}v{version}') from e
     return eprint.history, HTTPStatus.OK, {}
-    
\ No newline at end of file
+
+
+def get_eprint_pdf(identifier: str, version: int) -> Response:
+    """
+    Retrieve pdf for a specific e-print version.
+
+    Parameters
+    ----------
+    identifier : str
+        A valid arXiv identifier.
+    version : int
+        Numeric version of the e-print.
+
+    Raises
+    ------
+    :class:`.NotFound`
+        Raised when the requested identifier + version does not exist.
+
+    """
+    estore = CanonicalStore.current_session()
+    try:
+        eprint = estore.load_eprint(identifier, version)
+    except DoesNotExist as e:
+        raise NotFound(f'No such e-print: {identifier}v{version}') from e
+    return eprint.pdf, HTTPStatus.OK, {}
```

Comment:
Will want to update the return type annotation (see line 16). Easiest thing is probably use make the 0th element of the ``Response`` tuple a ``Union`` type with ``File``.

Reply:
I added a `FileResponse` type annotation - can you take a look? I've been a python 2.7 guy for a looooong time and haven't really done much python type annotating.

Reply:
Looks great! Typing will change your life...